src/lib/routes/admin-api/favorites.ts

```diff
@@ -48,7 +51,13 @@ export default class FavoritesController extends Controller {
                 openApiService.validPath({
                     tags: ['Features'],
                     operationId: 'removeFavoriteFeature',
-                    responses: { 200: emptyResponse },
+                    summary: 'Remove feature from favorites',
+                    description:
+                        'This endpoint removes the feature in the url from favorites',
+                    responses: {
+                        200: emptyResponse,
+                        ...getStandardResponses(401),
```

Comment:
Does it return a 404 if the feature doesn't exist?

```suggestion
                        ...getStandardResponses(401, 404),
```